pkg/virt-controller/watch/topology/filter.go

```diff
@@ -21,19 +21,7 @@ func IsSchedulable(node *v1.Node) bool {
 		return false
 	}
 
-	if node.Labels[virtv1.NodeSchedulable] != "true" || node.Spec.Unschedulable {
-		return false
-	}
-
-	if node.Spec.Taints != nil {
-		for _, taint := range node.Spec.Taints {
-			if taint.Effect == v1.TaintEffectNoSchedule {
-				return false
-			}
-		}
-	}
-
-	return true
+	return node.Labels[virtv1.NodeSchedulable] == "true"
```

Comment:
I just want to point out that this will be changed to false if e.g. the virt-handler daemonset needs to pull an image for a long time during update, or if there are other issues on virt-handler. As a consequence we may even exclude this node while there are actually VMs running on it.

Reply:
Thank you @rmohr, that's a good point!
Added, PTAL.